### DISCUSSION TITLE
Styled file input

### DISCUSSION THREAD

**Discussion by gshaw** [tailwindlabs/tailwindcss#3919] [Ideas]
I took inspiration from Bootstrap 5 purely CSS styled file input control and implemented it with Bootstrap.

Feels like something like this should be part of the Tailwind Forms plugin.  What do people think?
https://play.tailwindcss.com/TVHzmVU3k4?layout=horizontal

<img width="1283" alt="Screenshot 2021-04-02 at 7 16 58 PM" src="https://user-images.githubusercontent.com/33321/113465447-e21a4400-93e8-11eb-88ac-c0d5febe523f.png">

**Comment by coderinblack08** [comment]
Maybe move this to the form plugin repo?
